src/Day.js

```diff
@@ -43,10 +44,17 @@ export default function Day({
   if (empty) {
     return <div role="gridcell" aria-disabled className={className} />;
   }
+
+  const styles = Object.assign(
+    {},
+    ...Object.keys(modifiers).map(modifier => modifiersStyles[modifier])
+  );
+
```

Comment:
* Use `style` singular, not `styles` plural.
* Since `modifiersStyles` could be `undefined` (see my other comment), better write:

```
let style;
if (modifierStyles) {
// snip
```

just to reduce resource usage :)